apps/meeting/src/containers/DynamicMeetingControls/Styled.ts

```diff
@@ -4,11 +4,11 @@
 import styled from 'styled-components';
 
 interface StyledProps {
-  active: boolean;
+  $active: boolean;
```

Comment:
Change styled component props to use transient prop. 
- https://styled-components.com/docs/faqs#shouldforwardprop-is-no-longer-provided-by-default